File: quickstart-node/src/services/mail.service.ts
import { LoggerService } from '@services/monday-code';

const sendMail = (mailAddress: string, message: string): Promise<boolean> => {
  return new Promise((resolve, reject) => {
    try {
      // Put here send email logic
      LoggerService.getInstance().info(`Sending mail to ${mailAddress} with content: ${message}`);
      resolve(true);
    } catch (error) {
      reject(error);
    }
  });
};

export const MailService = {
  sendMail,
};

File: quickstart-python/services/mail_service.py
from services import LogsService


class MailService:
    @staticmethod
    def send_mail(mail_address: str, content: str):
        """
        Function that sends an email to the given address with the given content.
        """
        # Send email logic
        LogsService.info(f"Sending mail to {mail_address} with content: {content}",
                         payload={'content': content, 'mail_address': mail_address})
        pass
